src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp

```diff
@@ -25,6 +25,17 @@
 const Position SpawnPosition = {746.67f, 684.08f, 412.5f, 4.65f};
 #define CLEANUP_CHECK_INTERVAL  5000
 
+/*
+    Missing dialog/RP (already populated in DB) && spawns (can use ToC25 locations?) for:
+
```

Comment:
Could you add "@ todo: " ? It's for doxygen documentation.

Reply:
Doxygen won't recognize this. You need
```
/**
 * @todo: Something
 *
 *
*/